graphql/database/missions/UserMissionModel.js

```diff
@@ -13,17 +13,17 @@ class UserMission extends BaseModel {
   }
 
   static get hashKey() {
-    return 'missionId'
+    return 'userId'
```

Comment:
this makes sense, but we more often query and group UserMissions by missionId through the flow of the feature.  Also we would have to delete the tables in prod and recreate them.   I'd say let's keep the indexes as they are for now.

Reply:
The reason I did this is for permission to update the UserMissionModel (e.g. the `userIdMatchesHashKey` permission authorizer). It sounds like we may need to write a new authorizer?